tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -1,14 +1,12 @@
 """
 Integration Test for the bq module
 
-Ensures the create_dataset function has the proper parameters passed to it
+Ensures the define_dataset function has the proper parameters passed to it.
```

Comment:
nit `test_define_dataset()` can be part of the unit test too, it doesn't go talk to BigQuery, which is **great**.